tests/ui/app/reviews.js

```diff
@@ -132,7 +136,7 @@ casper.test.begin('Test review button hidden when logged in and cannot rate', {
             helpers.fake_login();
         });
 
-        helpers.waitForPageLoaded(function() {
+        waitForReviewsLoaded(function() {
```

Comment:
Actually it looks like `views.reload()` is going to run synchronously. So there isn't anything generic to wait for.